ProjectEureka/project-eureka-flutter
# PE-39 | Create the Profile Page

**What I Did:**
- Created the profile page with 3 different tabs for user's question, answer, and interests.
- Extracted the ListView from Home Page and made into a component for reuse in Profile Page.
- Linked "Edit Profile" button to ProfileOnboarding to allow users to edit their profile.
- Linked the "Create a New Question" on Home Page to the CreateNewQuestionPage.

**How To Test:**
- Run `flutter pub get`.
- From the Home Page, verify that everything is still working as it should.
- Using the Side Menu, switch to the profile page and verify that the UI is following to the specification of Figma.
- Switch between `Questions` `Answers` and `Interest` and verify that the data is changing.
- Verify that the `Edit Profile` button will allow the use to edit their profile and pop back the profile screen when done.
